web/sites/default/config/image.style.group_logo.yml

```diff
@@ -1,26 +1,15 @@
 uuid: b1bef4a7-044a-47cb-baf0-ae64889c0f99
 langcode: en
 status: true
-dependencies:
-  module:
-    - image_effects
+dependencies: {  }
 name: group_logo
 label: 'Group logo'
 effects:
-  920e5a1c-2d50-496b-8c18-5708f2ef4bc4:
-    uuid: 920e5a1c-2d50-496b-8c18-5708f2ef4bc4
+  397126e3-0887-4089-a10f-b7eca78a65ce:
+    uuid: 397126e3-0887-4089-a10f-b7eca78a65ce
     id: image_scale
     weight: 2
     data:
-      width: 80
-      height: 40
+      width: null
+      height: 72
       upscale: false
-  542b2d8a-9dc6-49d1-aa6e-f1484b7014a2:
```

Comment:
The current approach strips out the PNG optimization code that Greg added to all our image styles. That will lead to PNGs that look super fuzzy when uploaded as a PNG.

Reply:
After discussion we decided to discard implementing bitmap-based group logos in favor of SVGs. Some of the benefits to SVGs include pixel-perfect scaling, smaller file sizes, ability to modify styling of SVG using CSS if desired.
Part of this work is already captured in Jira with plans to groom.